packages/js-dpp/test/unit/identity/IdentityPublicKey.spec.js

```diff
@@ -118,6 +118,24 @@ describe('IdentityPublicKey', () => {
       expect(result).to.deep.equal(expectedHash);
     });
 
+    it('should return data in case ECDSA_HASH160', () => {
```

Comment:
Not sure I understand the point of this test

Reply:
in case of ECDSA_HASH160 we return data instead of data hash

Reply:
Oh... make sense